### ISSUE TITLE
Error when starting evcxr repl

### ISSUE THREAD

**Issue by StPanning** [intellij-rust/intellij-rust#5941]
* **IntelliJ Rust plugin version:** 0.3.128.3278-202
* **Rust toolchain version:** 1.45.1
* **IDE name and version:** CLion 2020.2
* **Operating system:** Linux 5.4.0-42-generic (Ubuntu Mate)

## Problem description
I get the following error, when I start the Rust REPL
```
/home/user/.cargo/bin/evcxr --ide-mode --disable-readline --opt 0
Welcome to evcxr. For help, type :help
No such file or directory (os error 2)

```
evcxr starts, but I have no Idea what causes the "no such file or directory" error

When I run `/home/user/.cargo/bin/evcxr --ide-mode --disable-readline --opt 0` from the command-line, I don't get an error message

## Steps to reproduce
Start the Rust REPL

**Comment by dima74** [comment]
@StPanning thanks for the report! Could you please provide [idea.log file](https://intellij-support.jetbrains.com/hc/en-us/articles/207241085-Locating-IDE-log-files)? And also `evcxr` version (`/home/user/.cargo/bin/evcxr --version`)

**Comment by StPanning** [comment]
@dima74 

hi,

First, the requested data:
Version: evcxr 0.5.3
[idea.log](https://github.com/intellij-rust/intellij-rust/files/5080344/idea.log)

Second:
Maybe it's a race condition.

When I run `strace -f -o clion.trace /home/murphy/bin/clion`

The error doesn't happen. (I am HeisenBug ;)

my hunch is, that strace slows parts of clion down enough, so some other process has time to finish.

**Comment by StPanning** [comment]
@dima74 the error also occurs everytime I press Ctrl+Enter

**Comment by StPanning** [comment]
@dima74 It also happens with IntelliJ IDEA Ultimate + Rust Plugin

**Comment by dima74** [comment]
@StPanning thanks for detailed information!

Unfortunately looks like there are no relevant exceptions in idea.log.

---

> the error also occurs everytime I press Ctrl+Enter

Do I understand correctly that it is possible to execute commands even after error? Could you please attach screenshot after executing few commands which should produce output (e.g. `println` or some arithmetic expression)?

---

Also could you try launch `/home/user/.cargo/bin/evcxr --ide-mode --disable-readline --opt 0` from [terminal inside IntelliJ](https://www.jetbrains.com/help/idea/terminal-emulator.html)?

**Comment by StPanning** [comment]
@dima74 

* First, Screenshot, after normal start of clion, with arithmetic expression:

![1p1](https://user-images.githubusercontent.com/1735049/90333678-24148800-dfc8-11ea-8582-32222909d57d.png)

* now 'straced' clion, everything works:
![strace_clion](https://user-images.githubusercontent.com/1735049/90333730-6b027d80-dfc8-11ea-9217-c2cb8fe3e0c7.png)

* now repl started inside intellj terminal:

 
![repl_intellj_terminal](https://user-images.githubusercontent.com/1735049/90333757-adc45580-dfc8-11ea-8626-af711f295c2c.png)

**Comment by StPanning** [comment]
@dima74 additional information:
I installed CLion on 
* the same machine under windows 10 (dualboot)
* my old notebook lenovo x220 (linux)

In both installations the repl works.

It seems my hardware + Ubuntu Mate + CLIon/Rust triggers a race-condition

**Comment by StPanning** [comment]
@dima74 
I was able to fix this problem.
It has nothing to do with the rust plugin.
It has nothing todo with strace
The reason was  that I used `.bashrc` extend the path variable.
I moved it to .profile. Now everything works.

As far as I traced the problem, this is what happens, if the path is declaered in the .bashrc
First `evcxr` is found in the path .
`evcxr` forks a child. this child doesn't find the cargo binary, and bails.

Thanks for your help. You helped to realize, that the problem was specific to my configuration and that made the debugging process way easier.
